### PR TITLE
Added login via plain text credentials

### PR DESCRIPTION
**Pull request by Kindrat**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -19,11 +19,23 @@ oc {
     // Set either a token file or the token itself. The tokenFile takes precedence, if it exists
     tokenFile = File("service-account.token")
     // token = "secret-token"
+    // To login via plain text credentials
+    credentials {
+        username = "admin"
+        password = "admin"
```
Comment (g3force) [inline Readme.md:25]: I suggest making at least the password parameter a closure. Then, you can either pass in a constant value, or you can do something like this (Groovy):
```
password = {
   return System.getenv("OC_PASSWORD")?.toCharArray()
}
```
That way, you can pass the password without risking committing a password accidentally.

**Review by g3force — COMMENTED** [review]
Just one suggestion, the rest looks could :+1:

File: Readme.md (hunk 1 above)
Comment (Kindrat) [reply Readme.md:25]: You can still do that inside of credentials action.

Comment (g3force) [reply Readme.md:25]: Oh, I see. Would you mind adding an example to the Readme? :) I think that would help other people.

Comment (Kindrat) [reply Readme.md:25]: Updated readme

Comment (g3force) [reply Readme.md:25]: thx

**Pull request merged**